config/checker/com.intellij.astub

```diff
@@ -225,6 +242,12 @@ interface Scheme {
 
 package com.intellij.openapi.progress;
 
+import static com.intellij.openapi.util.NlsContexts.ProgressText;
+
+interface ProgressIndicator {
+    void setText(@ProgressText @Untainted String text);
+}
+
```

Comment:
Actually... since you've already done the research on what params accept HTML text... I'd consider adding explicit `@Tainted` annotations for them.

CMIIW, but it's a shame from **soundess** PoV that the default for UNannotated fields is to assume `@Tainted`... although OTOH it'd be somewhat **impractical** if `@Untainted` was assumed for everything by default :/

Reply:
If you agree with this idea, I'd document the following distinction somewhere... maybe even in CONTRIBUTING.md to keep everything in a single place. 

And definitely, it needs to be documented that `@(Un)Tainted` annotations are used specifically to mark HTML usage... as they're pretty versatile, can be used for pretty much anything, whatever developers agree upon in the project

Reply:
> So yeah, what I'm effectively asking for is to distingiush b/w the cases of:

So you think about doing a similar job for places that can be @Tainted? to make it clear that HTML can be placed there

Reply:
Yes, indeed &mdash; just based on the research you've already put into a Google Doc, no need to do extra research as for now

Reply:
hmm I am stuck into a problem with kotlin files - it looks to me like checker annotations from java files are not applicable to kotlin classes ( e.g. giving too many arguments to the format method does not cause an error). Do you think that some configuration may change this behavior ?

Reply:
Yeah, Checker can't do anything about Kotlin code in general :/// we have to live with some potential unsoundness there :/

Reply:
For that reason we have e.g. sth like https://github.com/VirtusLab/git-machete-intellij-plugin/blob/master/scripts/prohibit-kotlin-undefined-property-keys ... as a low-quality _prosthetic_ for what Checker gives in Java files

Reply:
Huh remind me, both `label()` and `checkbox()` **forbid** the usage of HTML right?

Reply:
no `label()`, `checkbox()`, `button()` and  `com.intellij.ui.layout.ValidationInfoBuilder#error `allow html usage, but they are only used in kotlin files so I skipped adding them to stubs (I marked them in doc to write a checking script for them). Is it a good idea ?

Reply:
Yeah, makes sense indeed